src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/SchemaView.tsx

```diff
@@ -105,53 +110,40 @@ const SchemaView = (props: SchemaViewProps) => {
-                                                                                    qa
-                                                                                }
-                                                                                questionnaireResource={
-                                                                                    props.questionnaireResource
-                                                                                }
-                                                                                questionAndAnswerIndex={
-                                                                                    qaIndex
-                                                                                }
-                                                                            />
+                                                                            {qa.id.split(
+                                                                                '.',
+                                                                            )
+                                                                                .length ===
+                                                                                2 && (
+                                                                                <QuestionsAndAnswersDisplay
+                                                                                    questionAndAnswer={
+                                                                                        qa
+                                                                                    }
+                                                                                    questionnaireResource={
+                                                                                        props.questionnaireResource
+                                                                                    }
+                                                                                    questionAndAnswerIndex={
+                                                                                        qaIndex
+                                                                                    }
+                                                                                />
+                                                                            )}
+                                                                            {qa.id.split(
+                                                                                '.',
+                                                                            )
+                                                                                .length ===
+                                                                                3 && (
+                                                                                <SubQuestionsAndAnswersDisplay
```

Comment:
Should we consider removing one of the components? differences between styling for a sub equation and an equation may be done in the schema view.

Reply:
Yes, if the styling is similar you can just delete one of the components. Did only split it up to make it easier to style them